src/zeit/locales/de/LC_MESSAGES/zeit.cms.po

```diff
@@ -14,22 +14,22 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Development/Unknown\n"
-"POT-Creation-Date: Wed Nov 16 11:56:26 2016\n"
-"PO-Revision-Date: 2016-11-16 12:06+0100\n"
+"POT-Creation-Date: 2016-11-25 15:19+0100\n"
+"PO-Revision-Date: 2016-11-25 15:29+0100\n"
 "Last-Translator: Wolfgang Schnerring <wolfgang.schnerring@zeit.de>\n"
```

Comment:
@knutgoetz Hier sollte dein Name stehen, da du die Datei als letztes angefasst hast. Bitte trage deinen Namen in deinen Poedit-Einstellungen ein, damit das automatisch gemacht wird.